styles/components/_copy-to-clipboard-button.scss

```diff
@@ -0,0 +1,8 @@
+.cmp-copy-to-clipboard-button {
+  // If this button becomes more widely used, or with variable text,
+  // then this should instead use something like :before { content: attr(data-label) }
+  // to set the width such that changing the text within the button doens't change the
+  // width of the button.
+  width: 6.25rem;
```

Comment:
For flexibility, as noted in the comments, we might want to set `min-width` here instead of `width`. Or drop the width requirement entirely and use padding to make sure buttons look good regardless of the amount of text.

Reply:
Oh I think I've seen that trick before, something like this?

```css
&::before {
  content: 'Copy Text';
  visibility: hidden;
}

&::after {
  content: 'Copied!';
  visibility: hidden;
}
```

Yeah, that would work except for the fact that the text _could_ change if different props were passed in. I think `min-width` is probably the way the go, simple and effective 😄